src/app/Library/CrudPanel/CrudFilter.php

```diff
@@ -101,6 +101,9 @@ public function wasNotApplied()
      */
     public function apply($input = null)
     {
+        // before applying filters, store the base query count (query before any filters are applied)
+        $this->crud()->setOperationSetting('totalEntryCount', $this->crud()->getTotalEntryCount());
+
```

Comment:
Hmmm... why is this needed here @pxpm ? Seems out of place.

Reply:
(I think you've already explained it to me before 😅, but I forgot 🤦‍♂️)

Reply:
I replied to this question here https://github.com/Laravel-Backpack/CRUD/pull/4695#discussion_r986689198

Reply:
Hmm... ok... I understand why it's in ListOperation before we call `$this->crud->applyUnappliedFilters();` then... but... why is it here too? 

If we call it there... hasn't this already been done, why call it here too?